src/main/java/io/confluent/connect/elasticsearch/DataConverter.java

```diff
@@ -242,4 +276,12 @@ Object preProcessValue(Object value, Schema schema, Schema newSchema) {
         return value;
     }
   }
+
+  public enum BehaviorOnNullValues {
```

Comment:
not : May be sounds better as NullValueBehavior or NullValueHandlingStrategy

Reply:
Agreed. Have refactored to `NullValueBehavior`. Intentionally leaving the original name for `BEHAVIOR_ON_NULL_VALUES_CONF` though, since it seems to better match the idiom set up by other config name variables.

Reply:
I'm more comfortable with the enum matching the config. I'd prefer `BehaviorOnNullValues`. Also the important word is first in this version. `NullValuesBehaviour` kinda reads slowly to the actual meaning of the class. Up to you.

Reply:
+1 for `BehaviorOnNullValues`

Reply:
Alright, settling on `BehaviorOnNullValues` (in spite of a personal preference for `NullValueBehavior`) for now.